site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -116,10 +116,29 @@ Next, make sure that you can authenticate using gcloud:
 gcloud auth application-default login
 ```
 
-Now you can create your GKE cluster (optionally specifying the version of Agones you want to use):
+
+#### Creating the cluster in the default VPC 
+
+To create your GKE cluster in the default VPC just specify the project variable. 
+
+(optionally you can specify the version of Agones you want to use):
+
+```
+terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.7.0"]
 ```
-terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.0.0"]
+
+{{% feature publishVersion="1.7.0" %}}
```

Comment:
It would be better to put all of your additions gated behind the new version and leave the old text in place, with a `{{% feature expiryVersion="1.7.0" %}}` tag so that the live website doesn't change until the next version is cut.

Reply:
Agree, please make sure the shortcode is  `{{% feature expiryVersion="1.8.0" %}}`